datasets/assset/dataset.json

```
@@ -0,0 +1,266 @@
+{
+    "id": "assetBor",
+    "type": "dataset",
+    "auth": "FP/MWD",
+    "authorizationGrantor": "Verkeer en Openbare Ruimte, Gemeente Amsterdam",
+    "theme": [
+        "Openbare orde en veiligheid, Ruimte en topografie"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Verkeer en Openbare Ruimte, Gemeente Amsterdam",
+    "spatialDescription": "Gemeente Amsterdam",
+    "version": "1.0.0",
+    "title": "Beheerobjecten openbare ruimte",
+    "language": "nl",
+    "dateCreated": "2022-06-01T00:00:00+01:00",
+    "license": "Gesloten",
+    "accrualPeriodicity": "dagelijks",
+    "creator": "Datateam Beeldschoon/BOR",
+    "publisher": "Datateam Beeldschoon/BOR",
+    "description": "Objecten in de openbare ruimte die beheerd worden door de Gemeente Amsterdam",
+    "status": "beschikbaar",
+    "keywords": [
+        "asset",
+        "verhardingen",
+        "groenobjecten",
+        "terreindeel"
+    ],
+    "crs": "EPSG:28992",
+    "objective": "Het beschikbaar stellen van de gestructureerde statische objectgegevens van de te beheren objecten in de openbare ruimte volgens het Informatiemodel Beheer Openbare Ruimte (IMBOR).",
+    "temporalUnit": "",
+    "tables": [
+        {
+            "id": "verhardingen",
+            "type": "table",
+            "provenance": "asset_verhardingen",
+            "title": "Verhardingen",
+            "description": "Verharde lagen van een weglichaam, speel- en sportondergronden en onbegroeid terreindelen inclusief fundering.",
+            "version": "1.0.0",
+            "schema": {
+                "$schema": "http://json-schema.org/draft-07/schema#",
+                "type": "object",
+                "additionalProperties": false,
+                "mainGeometry": "geometrie",
+                "required": [
+                    "schema",
+                    "id"
+                ],
+                "display": "id",
+                "properties": {
+                    "schema": {
+                        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
```

We zijn over naar versie 1.2.0 van amsterdam-schema
```suggestion
                        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.2.0#/definitions/schema"
```